src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -142,8 +168,14 @@ trait SharedFileSystem {
       name -> adjusted
     }
 
-    // If this call is using Docker, adjust input paths, otherwise return unaltered input paths.
-    if (call.docker.isDefined) inputs map adjustPath else inputs
+    inputs map adjustPath
+  }
+
+  private def hashPath(originalPath: Path, size: Int = 8): String = {
+    val hash = MessageDigest.getInstance("MD5").digest(originalPath.toAbsolutePath.getParent.toString.getBytes) map { byte =>
+      f"$byte%02x"
+    } mkString ""
```

Comment:
@mcovarr Will probably start fetching his pitchfork on this comment but honestly this seems like one of those cases where I'd say just to damn the line length and put it all up on one